src/components/FilterGroup.tsx

```diff
@@ -119,7 +120,12 @@ function CheckboxOptions({
 }) {
   const searchUtilities = useSearchUtilities();
   const { searchValue } = useFilterGroupContext();
-  const shouldRenderOption = useCallback((option: { displayName: string, value: unknown }) => {
+  const shouldRenderOption = (option:
+  { displayName?: unknown, value: string | number | boolean | NumberRangeValue }) => {
```

Comment:
Instead of `unknown`, could we do `string | undefined` here? I can't remember off the top of my head, but we may also be able to do `displayName?: string`. The `string | undefined` may be redundant because of the optional nature of `displayName.